README.md

```diff
@@ -69,20 +69,24 @@ Or to specify the location explicitly:
 
 Use [Wraith](http://bbc-news.github.io/wraith/) ("A responsive screenshot
 comparison tool") to generate a visual diff to compare rendering changes in this
-application. Wraith has some dependencies you'll [need to install](http://bbc-news.github.io/wraith/os-install.html).
+application.
 
-First, on `master` branch, run:
+On the [Development VM](https://github.gds/gds/development) navigate to
+the project directory and run
 ```
-wraith history test/wraith/config.yaml
+sudo npm install --global phantomjs-prebuilt@2.1.7
```

Comment:
How about we add a `Gemfile` and `package.json` to `test/wraith` that allows us to explicitly declare these?